hugo/assets/scss/components/tabs-nav.scss

```diff
@@ -27,10 +28,27 @@
         }
     }
 
+    &__copy {
+        height: 36px;
+        position: relative;
+        width: 1rem;
+        z-index: 1;
+
+        &:focus,
+        &:hover {
+             pointer-events: cursor;
```

Comment:
cursor is not a valid value. should be pointer?  better to use button instead of a div faking to be a button